File: #168 Excel Sheet Column Title/168.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string s;
        while(n>0){
            int r = n%26;
            if(r) s.push_back(r+'A'-1);
            else s.push_back('Z'), n--;
            n /= 26;
        }
        std::reverse(s.begin(), s.end());
        return s;
    }
};

File: #168 Excel Sheet Column Title/168.py
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        ans = ""
        while n>0:
            r = n%26
            if r!=0: ans += chr(r+ord('A')-1)
            else:
                ans += 'Z'
                n -= 1
            n /= 26
        return ans[::-1]